lib/topics/state_and_behaviour.rb

```diff
@@ -2,4 +2,29 @@
 
 # exercise state and behaviour
 module StateAndBehaviour
+  # Describes car's state and behaviour
+  class Car
+    attr_accessor :year, :color, :model, :current_speed
+
+    def initialize(args)
+      @year = args[:year]   || 2000
+      @color = args[:color] || 'White'
+      @model = args[:model] || 'Mercedes W210'
+      @current_speed = 0
+    end
+
+    def self.default_car
```

Comment:
What about the [class structure](https://github.com/bbatsov/ruby-style-guide#consistent-classes)? Class methods have to be defined before instance methods